wsl/os/almalinux9/README.md

```diff
@@ -111,6 +111,9 @@ Using your preferred text editor, open `config.yml` and, under the `virtualhosts
 
 Already existing ones will be skipped, no need to comment or remove them.
 
+### NOTE
+> by using the pattern `*.localhost` for any new virtualhost, you do not need to modify `hosts` file in Win 11, are routed by default.
```

Comment:
```suggestion
> By using the pattern `*.localhost` for any new virtualhost, you do not need to modify the `hosts` file in Windows, because these are routed by default.
```
I'd not specify the OS version, in hope that this setup should work on upcoming version(s) as well. What do you think?